packages/lab/src/__tests__/__e2e__/calendar/Calendar.cy.tsx

```diff
@@ -259,35 +247,45 @@ describe("GIVEN a Calendar", () => {
             }).focus();
             cy.findByRole("button", {
               name: adapter.format(testDate, "DD MMMM YYYY"),
-            }).should("be.focused");
+            })
+              .invoke("attr", "class")
+              .should("match", /saltCalendarDay-focused/);
```

Comment:
Why did testing real focus not work?

Reply:
added additional check to ensure it is focused as well.